python_scripts/trees_regression.py

```diff
@@ -36,6 +36,12 @@
                                    data_train[data_columns[0]].max()),
                          columns=data_columns)
 
+# %% [markdown]
+# Using the term "test" here refers to data that was not used for training.
+# It should not be confused with data coming from a train-test split, as it
+# was generated in equally-spaced intervals for the visual evaluation of the
+# predictions.
+
```

Comment:
Maybe we can repeat the note that I suggested above here. Or just put it here since this section will appear first in the chronological order of the MOOC. As you prefer.

Reply:
I added the suggestion here and just added a recall of this message in the bagging notebook